lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -1180,6 +1181,36 @@ describe('workers/repository/process/lookup/index', () => {
       );
     });
 
+    it('should warn if no digest could be found but pinning is enabled', async () => {
+      config.currentValue = 'v1.0.0';
+      config.currentDigest = 'bla';
+      config.digestOneAndOnly = true;
+      config.depName = 'angular/angular';
+      config.pinDigests = true;
+      config.datasource = GithubTagsDatasource.id;
+
+      // Only mock calls once so that the second invocation results in
+      // no digest being computable.
+      jest.spyOn(githubGraphql, 'queryReleases').mockResolvedValueOnce([]);
+      jest.spyOn(githubGraphql, 'queryTags').mockResolvedValueOnce([
+        {
+          version: 'v2.0.0',
+          gitRef: 'v2.0.0',
+          releaseTimestamp: '2022-01-01',
+          hash: 'abc',
+        },
+      ]);
+
+      const res = await lookup.lookupUpdates(config);
+      expect(res.updates).toMatchSnapshot();
```

Comment:
no snapshots, read our best practices in development docs

Reply:
took it from the test above, but can be removed anyway.